scania-digital-design-system/sdds
# Feature/link

- Add link style on <a> element and class .sdds-link
- Disable underline on button element that is used with <a class='sdds-btn...>

How to test:
- See storybook and change option for disable to see disable state
- Figma >> UI Kit (Global) >> Link or Figma >> Components >> Link
